samza-rest/src/main/java/org/apache/samza/monitor/LocalStoreMonitorConfig.java

```diff
@@ -46,7 +46,7 @@
    * Defines the comma separated list of job status servers of the form
    * "Host1:Port1,Host2:Port2".
    */
-  private static final String CONFIG_JOB_STATUS_SERVERS = "job.status.servers";
+  private static final String CONFIG_JOB_STATUS_SERVERS = "jobs.status.servers";
```

Comment:
That's probably the best option. LocalStoreMonitor is newer than JobsResource, so it has had less exposure and will have less/no impact on users.